aten/src/ATen/core/Formatting.cpp

```diff
@@ -238,10 +238,15 @@ std::ostream& print(std::ostream& stream, const Tensor & tensor_, int64_t linesi
     stream << "size:\n" << tensor_.sizes() << "\n";
     stream << "]";
   } else {
-    Tensor tensor = tensor_.to(kCPU, kDouble).contiguous();
+    Tensor tensor;
+    if (tensor_.is_quantized()) {
+      Tensor tensor = tensor_.dequantize().to(kCPU, kDouble).contiguous();
```

Comment:
I hit this bug today. so we have never actually printed a quantized Tensor before...